charts/datadog/values.yaml

```diff
@@ -53,6 +53,16 @@ datadog:
     # datadog.secretBackend.timeout -- Configure the secret backend command timeout in seconds.
     timeout:  # 30
 
+    # datadog.secretBackend.enableGlobalPermissions -- Whether to create a global permission allowing Datadog agents to read all secrets
+    enableGlobalPermissions: true
+
+    # datadog.secretBackend.roles -- Creates roles for Datadog to read the specified secrets
```

Comment:
FYI - In its current form, you can define `roles` **and** `enableGlobalPermissions: true`, there is no conditional being performed on the two of them. Unsure if by stating your suggestion in the docs it will cause some confusion

Reply:
All good with the latest changes, thanks!